plugins/factoids.py

```diff
@@ -300,6 +301,25 @@ async def tag_top(self, ctx: Context) -> None:
             results = list(await session.execute(stmt))
             await ctx.send("\n".join(format("{!i}: {} uses", conf.prefix + name, uses)
                 for name, uses in results))
+            
+    @tag_command.command("showhelp")
+    async def tag_showhelp(self, ctx: Context) -> None:
+        """Show factoids with the 'help' flag."""
+        async with sessionmaker() as session:
+            aliases = aliased(Alias)
+            stmt = (select(Alias)
+                .join(Alias.factoid)
+                .where (Factoid.flags["help"].astext.cast(Boolean) == True)
```

Comment:
This feels like a bodge - probably there's a better way to do this. An alternative is to select all unique factoids then print (lines 320-321) only those with the flag. I'm also unsure of which is more efficient.